src/Provider/Provider.Web/Views/Dashboard/Dashboard.cshtml

```diff
@@ -119,7 +132,7 @@
         $(function() {
             $("#filter-submit").hide();
 
-            $("#filter").on("change",
+            $("#Filter").on("change",
```

Comment:
Can you use something like this to workout the Filter Id please. Then it won't break should we rename the property.
`var filterId = "#@Html.IdFor(m => m.Filter)";`